### PR TITLE
Sdk-proofreading

### PR DESCRIPTION
**Pull request by denisvitchevsky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
+This repository contains the documentation for creating Wilco quests using the Wilco Editor SDK. If you don't have a creator account yet, please do the following:
+
+Existing Wilco users: Click here to become a creator.
+
+Not a user yet? Click here to sign up as a creator.
 
 ### **Getting started with the Wilco Editor SDK**
-This is the Editor SDK documentation repository outlining how to create a quest from scratch using example quests. <br />
-To get started please go to https://github.com/trywilco/wilcosdk/blob/main/questcreation.md
+The following repository outlines how to create a quest from scratch, including examples.
```
Comment (karenuriah5) [inline README.md:13]: The following repository outlines how to create a quest from scratch, or by using an example quest template.

**Review by denisvitchevsky — COMMENTED** [review]
More edits

File: questcreation.md [hunk 2]
```diff
@@ -1,42 +1,41 @@
 # Quests Creation
 
-### Terminology
+### **Terminology**
 
-- **Quest:** A series of steps, all connected together based on their topic.
-- **Step:** A small task inside a quest. Usually, a step will represent a single user interaction.
-- **Trigger:** An event in the system caused by the user. e.g., New chat message, PR opened, env var added to Heroku, etc.
-- **Action:** A reaction to a triggered event that causes the user’s state in the quest to change. e.g., advance to next step, approve/reject user’s PR, send a message from bot, etc.
+- **Step:** A task that usually represents a single user interaction.
```
Comment (karenuriah5) [inline questcreation.md:5]: Logically, I don't think we should change the order between Quests and Steps. Quest is the higher level of hierarchy.

Comment (denisvitchevsky) [reply questcreation.md:5]: Yes, but we explain what a quest is by defining it through the lower hierarchy. In terms of comprehension, they first need to understand what a step is.

File: questcreation.md [hunk 3]
```diff
@@ -1,42 +1,41 @@
...
 - **FlowNode:** when a step starts or triggers are recognized, a Flow will begin. A single element in the flow is called FlowNode. A node is a combination of actions and conditions.
 - **GitHub Actions:** command executed as part of GitHub workflow during PR checks.
 
 ### **How to Create a Quest**
 
-1. Create a new repository by clicking on "Use this template" on the [quest template](https://github.com/trywilco/quest-template). This is a clone of the Funnel Drop quest and is fully playable as-is.
-2. Update the quest files to support your new quest, according to the [Quest Development Guidelines](https://github.com/trywilco/wilcosdk/tree/main/Quests%20Creation/Quest%20Development%20Guidelines):
+1. Create a new repository by going to the [quest template](https://github.com/trywilco/quest-template) and clicking the "use this template" button. This template a clone of the ["Funnel Drop" quest](https://app.wilco.gg/catalog/quest/mobile-responsiveness) and is fully playable as-is.
```
Comment (karenuriah5) [inline questcreation.md:16]: "This template **is** a clone of..."

File: Quests Creation/Quest Configuration Files.md [hunk 4]
```diff
@@ -95,11 +95,11 @@ Supported images extensions:
 
 The following files must be placed in the assets folder:
 
-- **cover:** Main cover image of the quest a presented in the catalog of user’s quests page.
+- **cover:** The main cover image of the quest, shown in the catalog.
```
Comment (karenuriah5) [inline Quests Creation/Quest Configuration Files.md:98]: Please add: The name of this file must be Cover.

File: Quests Creation/Quest Configuration Files.md [hunk 5]
```diff
@@ -95,11 +95,11 @@ Supported images extensions:
...
     
     ![Screen Shot 2022-09-06 at 23.53.34.png](Quest%20Configuration%20Files/Screen_Shot_2022-09-06_at_23.53.34.png)
     
-- **logo**: Logo image presented when selecting a quest from the catalog.
+- **logo**: The logo image presented when selecting a quest from the catalog.
```
Comment (karenuriah5) [inline Quests Creation/Quest Configuration Files.md:102]: Please add: The name of this file must be Logo.

File: Quests Creation/Quest Resources.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # Quest Resources
 
-A quest might be dependent on resources being functional in order for the user to start the quest. In many cases, a quest might require configuration to be removed. An example for that is when user plays the same quest again and need to clear any configurations added by the user in the quest itself.
+A quest might be dependent on resources being functional before a user can start the quest. In many cases, a quest might require configuration to be removed. An example for that is when user plays the same quest again and needs to clear any configurations added during the previous playthrough.
```
Comment (karenuriah5) [inline Quests Creation/Quest Resources.md:3]: "An example for that is when **the** user plays the..."

File: Quests Creation/Triggers and Payload.md [hunk 7]
```diff
@@ -26,43 +26,43 @@ trigger:
     ...
 ```
 
-Payload:
+Generated payload:
 
 - `userMessageText`: the text user wrote to the bot
```
Comment (karenuriah5) [inline Quests Creation/Triggers and Payload.md:31]: "....the text **the** user wrote to the bot"

**Review by karenuriah5 — APPROVED** [review]

**Pull request merged**